website/docs/concepts/html/elements.mdx

```diff
@@ -16,30 +16,21 @@ used as a `Html` value using `VRef`:
 
 ```rust
 use web_sys::{Element, Node};
-use yew::{Component, Context, html, Html};
+use yew::prelude::*;
 use gloo::utils::document;
 
-struct Comp;
-
-impl Component for Comp {
-    type Message = ();
-    type Properties = ();
-
-    fn create(_ctx: &Context<Self>) -> Self {
-        Self
-    }
-
-    fn view(&self, _ctx: &Context<Self>) -> Html {
-        // Create a div element from the document
-        let div: Element = document().create_element("div").unwrap();
-        // Add content, classes etc.
-        div.set_inner_html("Hello, World!");
-        // Convert Element into a Node
-        let node: Node = div.into();
-        // Return that Node as a Html value
-        Html::VRef(node)
-    }
+#[function_component]
+fn MyComponent() -> Html {
+    // Create a div element from the document
+    let div: Element = document().create_element("div").unwrap();
+    // Add content, classes etc.
+    div.set_inner_html("Hello, World!");
+    // Convert Element into a Node
+    let node: Node = div.into();
+    // Return that Node as a Html value
+    Html::VRef(node)
```

Comment:
This creates a new `Element` on each rerender, which works fine here because it never rerenders, but if you copy-paste this example into a larger component, might lead to surprising results. At least a warning is warranted, or you can maybe show and explain a use-case of the `use_state` hook?

Reply:
This seems like a great use case for use_memo hook. There should also be a warning that this component will fail to render with SSR